src/statmodels.jl

```diff
@@ -145,11 +145,18 @@ end
 
 const adjr² = adjr2
 
+
 abstract RegressionModel <: StatisticalModel
 
 fitted(obj::RegressionModel) = error("fitted is not defined for $(typeof(obj)).")
 model_response(obj::RegressionModel) = error("model_response is not defined for $(typeof(obj)).")
 residuals(obj::RegressionModel) = error("residuals is not defined for $(typeof(obj)).")
+
+"""
+    predict(obj::RegressionModel [, newX])
```

Comment:
We've done the docstrings in this package slightly differently. IIRC it's not entirely consistent in Base.